src/recon.rs

```diff
@@ -2660,6 +2656,10 @@ pub(crate) unsafe fn rav1d_recon_b_intra<BD: BitDepth>(
                         } else {
                             None
                         };
+                        let interintra_edge =
+                            BD::select_mut(&mut t.scratch.c2rust_unnamed_0.interintra_edge);
+                        let edge_array = &mut interintra_edge.0.edge;
+                        let edge_offset = 128 as usize;
```

Comment:
```suggestion
                        let edge_offset = 128;
```

If it's a literal, rustc should be able to figure out it's a `usize`.  If it's ambiguous, though, things are fine as is, though `128usize` can also work.